backend/django/core/views/api_annotate.py

```diff
@@ -159,8 +159,9 @@ def skip_data(request, data_pk):
             process_irr_label(data, None)
     else:
         # the data is not IRR so treat it as normal
-        move_skipped_to_admin_queue(data, profile, project)
-        createUnresolvedAdjudicateMessage(project, data, request.data["message"])
+        if request.data["addToAdminQueue"]:
```

Comment:
why is this added here? Doe that mean if data is skipped using the skip button and is IRR then it is send through the administrator skip system normally? I'm not sure I agree with that, since I thought the skip button mean "I want to look at this later" which is not the same as "I think this is uncodable"